app/controllers/users_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
```

Comment:
Please remove commented code from here because it can make the whole code unclean.